### PR TITLE
fix: refresh UI organizations after creating organization

### LINKED ISSUE #2359: Newly created org not visible
A newly created organization is not visible in organization dropdown menus when creating a new resource, licence, etc. Browser refresh or logout/login is needed for it to appear.

### PR DESCRIPTION
**Pull request by opqdonut**
fixes #2359

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue

## Documentation
- [x] update changelog if necessary

## Testing
- [x] valuable features are integration / browser / acceptance tested automatically

## Follow-up
- [x] no critical TODOs left to implement

### CODE REVIEW

File: src/cljs/rems/config.cljs [hunk 1]
```diff
@@ -24,3 +27,39 @@
 (defn dev-environment? []
   (let [config @(rf/subscribe [::config])]
     (boolean (:dev config))))
+
+;; organizations
```
Comment (opqdonut) [inline src/cljs/rems/config.cljs:31]: should we also move fetching themes and translations to this file?

Comment (Macroz) [reply src/cljs/rems/config.cljs:31]: Could be but maybe it also should be named after the APIs i.e. public? Or is that wrong too? Organizations are not exactly config so they could be in their own ns. They are also not public.

File: src/cljs/rems/administration/organizations.cljs [hunk 2]
```diff
@@ -21,6 +22,11 @@
  ::fetch-organizations
  (fn [db]
    (let [description [text :t.administration/organizations]]
+     ;; Refresh :organizations subscription. We do this here
+     ;; since it would be surprising if an organization was visible on
+     ;; this page but not selectable e.g. when creating a resource.
+     (config/fetch-organizations!)
```
Comment (Macroz) [inline src/cljs/rems/administration/organizations.cljs:28]: I think this is a bit odd thing to do. It's not strictly necessary after the previous change?

**Review by Macroz — APPROVED** [review]

File: src/cljs/rems/administration/organizations.cljs (hunk 2 above)
Comment (opqdonut) [reply src/cljs/rems/administration/organizations.cljs:28]: I think it's useful if e.g. you've created an org using the API and then want to use it via the UI, or perhaps somebody else just created an org and tells you to check it out.

**Pull request merged**